### ISSUE TITLE
Exception in episode 1

### ISSUE THREAD

**Issue by markgdawson** [M4rkD/python-OOP-novice#10]
Mixing the concepts of exception and inheritance could be confusing here. Could introduce inheritance first, then introduce the use case of exception as a use of that.

Something like:
```
class MyArray(numpy.ndarray):
  pass
```

followed by
```
a = MyArray(shape=(10,10))
type(a) # shows MyArray
isinstance(a, MyArray)
isinstance(a, np.ndarray)
```

Would be very nice if we could find an example like this which could be extended (i.e. by adding a body to the class) in later episodes?

**Comment by edbennett** [comment]
I agree that it would be good to disentangle these two points. The only reason I used exceptions here is because it's the main place in the Python standard library (that I could find) where inheritance was used significantly, and in ways that could be easily demonstrated and be found useful.

My concern with the proposed fix is that now we have introduced both class definitions and subclass definitions in a single line, whereas the aim of this episode was to use existing classes but not define any fresh ones. We'd be replacing one mix of concepts with another.
